aries-test-harness/features/0011-0183-revocation.feature

```diff
@@ -22,7 +22,7 @@ Feature: RFC 0011 0183 Aries agent credential revocation and revocation notifica
          | issuer | credential_data   | request_for_proof              | presentation                  |
          | Acme   | Data_DL_MaxValues | proof_request_DL_revoc_address | presentation_DL_revoc_address |
 
-   @T001.1-RFC0011 @RFC0011 @P2 @normal @AcceptanceTest @Schema_DriversLicense_Revoc @Indy @DIDExchangeConnection
+   @T001.1-RFC0011 @RFC0011 @normal @AcceptanceTest @Schema_DriversLicense_Revoc @Indy @DIDExchangeConnection
```

Comment:
Questions -- as I understand this test, it's got an issue and a verify, so shouldn't it include RFC0036 and RFC0037?

Also, it has revocation, so shouldn't we have "@revocation", so that we can easily exclude all revocation tests?  I called that one out as one I would like to be able to run alone (e.g. "-t @revocation") or exclude (e.g. "-t @RFC0036 -t ~@revocation").

I'm not sure why RFC0011 is called out for that one -- attachments.  Attachments are used in many tests independent of what is used -- e.g. in any test using a VC.  Perhaps my question more specifically is why it would be associated as the "primary" RFC (eg. included with the T001 tag).

Also, wouldn't the @DIDExchangeConnection be just replaced with an "@RFC0023"?

Reply:
That is a good point on including RFC0036 and RFC0037. I've left the RFC tag represent the main goal of the test scenario, which is revocation. The issue and verify are used to accomplish the goal. If we do add them in, we should distinguish them from the main goal. I am also considering how to capture the RFCs that are indirectly tested because they are contained or used in the protocol under test. So there is the Main:RFC0011, Secondary:RCF0036, Secondary:RFC0037, & Indirect:RFCyadayada.   This isn't simplifying the tags though. 

I had thought RFC0011 was the Revocation RFC, but not looking at it I was wrong. Not sure how that happened. What is the Revocation RFC number? That would be used instead of the `revocation` tag.

on the DIDExchangeConnection tag, that is a code controller. The RFC number would mean  (at my current understanding of the tags) that RFC0023 is the goal of the test. It isn't, it is like 0036 and 0037 above.

Reply:
I've always wondered by RFC0011 was referenced :-).  It confused me.

There is no revocation RFC.  There is a revocation notification RFC that will be in AIP 2.0 (that 0183), but 0011 is decorators.  As such, I would go with "revocation" vs. an RFC.

I would be against the "Main:" and "Secondary:" ideas because I want to use them to be able to run all the tests for a specific RFC.  We already have a mechanism to say what the main one is (in the ID tag), so I don't see why we need to specify further.

Reply:
Indy HIPE 0011 is revocation, maybe that's where it comes from?

https://hyperledger-indy.readthedocs.io/projects/hipe/en/latest/text/0011-cred-revocation/README.html

Reply:
Thanks Timo, I think that is exactly where I got the 0011 number. 

Should I prefix 0011 with something else, or just use RFC0183 as the reference? 

Should I add AIP20 for revocation? Right now it doesn't have an AIP assigned because it wasn't part of anything.

Reply:
Since the HIPE is Indy specific, and revocation is not, I would not recommend using RFC0011 (different RFC) or HIPE0011 (Indy-specific). For now, I think it best to just use "revocation" as the indicator.

For the AIP references, I suggest that AIP10 be used of the credential exchange protocols are RFC 0036/0037, and AIP20 if the protocols are RFC 0453/0454.

That allows, for example, an test agent to run tests for "AIP10" and not "revocation" if it doesn't support it.

Reply:
RFC0011 changed to @revocation. Test ids are changed to T001-HIPE0011